tailwindcss.config.js

```diff
@@ -7,27 +7,154 @@ module.exports = {
     themes: [
       {
         mytheme: {
-        
-      "primary": "red",
-              
-      "secondary": "#F000B8",
-              
-      "accent": "#37CDBE",
-              
-      "neutral": "#3D4451",
-              
-      "base-100": "#FFFFFF",
-              
-      "info": "#3ABFF8",
-              
-      "success": "#36D399",
-              
-      "warning": "#FBBD23",
-              
-      "error": "#F87272",
+          "primary": "red",
+          "secondary": "#F000B8",
+          "accent": "#37CDBE",
+          "neutral": "#3D4451",
+          "base-100": "#FFFFFF",
+          "info": "#3ABFF8",
+          "success": "#36D399",
+          "warning": "#FBBD23",
+          "error": "#F87272",
+          // denali rainbow colors
+          "grey": {
```

Comment:
Should I prefix these with "denali" so that they don't conflict with the default Tailwind colors ([https://tailwindcss.com/docs/customizing-colors](https://tailwindcss.com/docs/customizing-colors))?